ui/lib/apps/Statement/pages/List/index.tsx

```diff
@@ -70,11 +70,29 @@ export default function StatementsOverview() {
     allStmtTypes,
     loadingStatements,
     tableColumns,
+    isTimeRangeOutdated,
 
     downloadCSV,
     downloading,
   } = controller
 
+  const [showTimeRangeOutdatedTip, setShowTimeRangeOutdatedTip] = useState(
+    false
+  )
+  useEffect(() => {
+    let timer
+    if (isTimeRangeOutdated) {
+      setShowTimeRangeOutdatedTip(true)
+      timer = setTimeout(() => {
+        setShowTimeRangeOutdatedTip(false)
+      }, 3000)
```

Comment:
Maybe it would be simpler to just keep it, and remove it when refresh is clicked. Otherwise user will no longer see the tooltip any more?